server/middleware/party.middleware.js

```diff
@@ -43,7 +44,14 @@ const middleware = {
       });
     }
 
-    const { rows: newParty } = await db(queries.updatePartyName(req.body.name, id));
+    if (partyName.length) {
+      return res.status(409).json({
+        status: 409,
+        error: 'party already present',
```

Comment:
I think you should make this message more descriptive. It should tell the user what in the party already exists. Something like
```suggestion
        error: 'Party name already exists',
```